### DISCUSSION TITLE
Monero to fiat trades using BTC as the multi-sig

### DISCUSSION THREAD

**Discussion by pazza83** [bisq-network/bisq#5285] [Ideas]
XMR/BTC is the most liquid market on Bisq and generates most of the revenues.

XMR/Fiat would be popular and is commonly asked about for example: this [Reddit post](https://reddit.com/r/bisq/comments/lzmc64/why_cant_we_buy_monero_directly_with_fiat/)

On Monero communities people are also frequently asking about how to buy XMR using fiat and get referred to a centralized exchange such as [Kraken](https://www.kraken.com/en-gb/learn/buy-monero-xmr) or to a P2P marketplace such as [Local Monero](https://localmonero.co)

Bisq is developed so that BTC needs to be one of the trading pairs. There seems to have been some discussion about adding XMR as a trading pair but whilst their was general agreement that this would be good, the resources needed to develop it made it low on list of priorities.

## Idea

My idea is to do a half way house to see how popular XMR/Fiat would be.

Take the most 2 liquid markets for fiat in Bisq USD and EUR and give them XMR/USD and XMR/EUR pairs respectively.

BTC can still be used in the multisig. 

Sellers of XMR would deposit 115% BTC to cover trade amount and security deposit. When trade complete seller would receive their full security deposit back.

Buyers of XMR would deposit 15% BTC to cover security deposit. When trade complete buyer would receive their full security deposit back.

## Benefits

The benefits to trading XMR/fiat could be as follows:

- Introduces a new revenue source for Bisq, traders that would otherwise utalize the above two platforms or similar. 
- Allow Bisq to experiment with XMR/USD and XMR/EUR pairs with reduced dev resources.
- If they proved popular then the above could potentially be developed to provide XMR as a base pair.

Let me know your thoughts.

### Edit posted on Reddit on 02.04.21:

[Would you like to be able to use Bisq to buy Monero directly with fiat?](https://www.reddit.com/r/Monero/comments/micevf/would_you_like_to_be_able_to_use_bisq_to_buy/)

Monero community seems very supportive of this implementation. 

### Edit 04.04.21:

I have opened a project for this here: https://github.com/bisq-network/projects/issues/50 please continue any further discussion here.

**Comment by Bl5ckj5ck** [comment]
I totally support this idea. Opening the xmr/usd-xmr/eur would satisfy many users’ requests. Bisq’s volume shows that users want to buy xmr and unfortunately their only current solution is Localmonero (which is not completely decentralized). 
Opening these 2 markets would let users who are interested in both btc and xmr operate directly through just one single platform which is Bisq

**Reply by pazza83** [reply]
Thanks for the feedback. I agree a decentralized option for trading fiat / Monero seems like it would be popular. Might also bring new users to Bisq.

**Comment by MwithM** [comment]
I understand that BTC would be used only as collateral, and XMR and fiat would need to be exchanged in their own way.
Who would send first? I suppose XMR buyer should send first, as buyer has more into deposit tx.
After XMR seller sends XMR to XMR buyer (happy ending), 115% BTC used as collateral should go back entirely to seller.
I'm not sure that this amounts used as collateral will work properly, game theory cases need further study.
This method could be used to trade anything.

**Reply by pazza83** [reply]
Hi @MwithM 

Thanks for the comments. Not sure if game theory would have a factor in these trades (at least any more so that a normal BTC/fiat trade). Let me know if I have missed anything:

Here is how I think the trade would work:

**Sell XMR**

Example for 10 XMR trade (€1,900)

- Sell XMR / Buy Fiat 
- XMR seller makes an offer to sell 10 XMR for €1,900
- XMR seller would deposit 115% of trade amount in BTC (0.046 BTC)
- When XMR buyer accepts offer they would deposit 15% of trade amount in BTC (0.006 BTC)
- 0.052 BTC locked in multi-sig
-  XMR buyer sends €1,900 to XMR Seller's bank account
- XMR seller confirms receipt of €1,900, sends 10 XMR
- XMR buyer confirms 10 XMR have been received  (or trade auto-confirmed)
- BTC deposit is released to buyer and seller
- If trade goes wrong it can enter mediation  

**Buy XMR**

Example for 10 XMR trade (€1,900)

- Buy XMR / Sell Fiat 
- XMR buyer makes an offer to buy 10 XMR for €1,900
- XMR buyer would deposit 15% of trade amount in BTC (0.006 BTC)
- When XMR seller accepts offer they would deposit 115% of trade amount in BTC (0.046 BTC)
- 0.052 BTC locked in multi-sig
- XMR buyer sends €1,900 to XMR Seller's bank account
- XMR seller confirms receipt of €1,900, sends 10 XMR
- XMR buyer confirms 10 XMR have been received  (or trade auto-confirmed)
- BTC deposit is released to buyer and seller
- If trade goes wrong it can enter mediation  

>This method could be used to trade anything.

Yes BTC acts a great collateral 😃

I think in future it could be combined with private 1 to 1 trades like suggested here: https://github.com/bisq-network/bisq/discussions/5314

I see lots of potential for trades where users would like an alternative to a traditional escrow that is decentralized and anonymous. 

For now though I would like to give other P2P platforms for users of Monero a run for their money. Lots of sellers on other P2P platforms are able to KYC users. Having a platform for trading with no KYC would be popular. I see this a gateway to onboarding more Monero traders who are not fans of KYC to Bisq.

**Reply by geonic1** [reply]
Why not allow the use of XMR or BTC as collateral? Many will prefer to avoid the $20+ network fee if they can help it.

**Reply by pazza83** [reply]
Hi @geonic1 thanks for the comment, it would be great to use XMR as the collateral however this would require major changes to the trade protocol and eat up dev resources. The idea would be to see if their is a market for fiat <-> XMR before taking on such a large commitment.

**Reply by geonic1** [reply]
Great! Didn't realize that. Let's take it one step at a time then. If there is a competent developer who could deliver this and there is a lack of funding, there seems to be enough community interest in this idea that a Monero CCS proposal might be warranted.

**Comment by pazza83** [comment]
I have opened the project here: https://github.com/bisq-network/projects/issues/50

Please use this link for any further discussion.

**Comment by Conza88** [comment]
> Let me know your thoughts.

Sure. But not higher priority than any of the fundamental changes required for BTC / Bisq. #5271 

1. API Release
4. Reduce trade protocol to 1 single transaction
5. Increase liquidity by allowing each offer to have multiple payment methods & currencies associated with it
6. Implement new-user-onboarding and new UI design
8. Improve BSQ buying/selling overall process
